test/Assig1/ValidatorTest.java

```diff
@@ -0,0 +1,144 @@
+package Assig1;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import assignmentone.exceptions.InvalidInputError;
+import assignmentone.model.Item;
+import assignmentone.utils.ValidatorUtils;
+import org.junit.jupiter.api.Test;
+
+/*
+ * This class tests Assignment 1 on different testcase
+ */
+class ValidatorTest {
+
+    private ValidatorUtils validator;
+
+    /*
+     * This test method is used to check whether correct input is passing test or not
+     */
+    @Test
+    void devTest() throws Exception {
+
+        String[] args = new String[]{"-name", "snehil", "-type", "raw", "-price", "123", "-quantity", "2"};
+        validator = new ValidatorUtils();
```

Comment:
util class ? why are u declaring it again

Reply:
ok